.travis.yml

```diff
@@ -18,13 +28,9 @@ install:
   - source activate test-environment
   - python setup.py install
   - pip install --upgrade pytest
-
+  - if [[ "$IMAGE_BACKEND" == "Pillow-SIMD" ]]; then
+      pip uninstall -y pillow && CC="cc -mavx2" pip install --force-reinstall pillow-simd;
```

Comment:
shouldn't `-mavx2` be `-mnative`. I dont think travis nodes are guaranteed to have AVX2

Reply:
This is the suggestion from pillow-simd github on how to install if If we'd like to install the AVX2-enabled version. I can change it with `-mnative` or just remote.